data/src/main/kotlin/com/vultisig/wallet/data/api/models/KyberJson.kt

```diff
@@ -0,0 +1,108 @@
+package com.vultisig.wallet.data.api.models
+
+import com.vultisig.wallet.data.models.Chain
+import com.vultisig.wallet.data.models.Coin
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
+
+sealed class KyberSwapError(message: String) : Exception(message) {
+    class ApiError(val code: Int, message: String, val details: List<String>?) :
+        KyberSwapError(message)
+
+    class TransactionWillRevert(message: String) : KyberSwapError(message)
+    class InsufficientAllowance(message: String) : KyberSwapError(message)
+    class InsufficientFunds(message: String) : KyberSwapError(message)
+}
```

Comment:
these types are not used anywhere. Was the purpose to have specific UI? isn't just generic exceptio with message enough?

Can we just use bodyOrThrow() which will contains the message?